qkeras/autoqkeras/autoqkeras_internal.py

```diff
@@ -165,20 +164,24 @@ def __init__(
     else:
       self.quantization_config = quantization_config
 
-  def _adjust_limit(self, name, default):
+  def _adjust_limit(self, default):
     """Makes sure limit has all the fields required."""
-
     if isinstance(default, list):
-      assert len(default) == 3
+      assert 3 <= len(default) <= 4 
     else:
       default = [default] * 3
 
     # we consider that if name is not there, we will ignore the layer
-    if name in self.limit:
-      length = len(self.limit[name])
-      # for Activation we only need one entry.
-      if length < 3 and name != "Activation":
-        self.limit[name] = self.limit[name] + default[length:]
+    for name in REGISTERED_LAYERS:
+      if name in self.limit:
+        length = len(self.limit[name])
+        if length < 4 and name in ["SimpleRNN", "LSTM", "GRU", "Bidirectional"]:
```

Comment:
Jose, could you use a constant variable to replace this here. So that we can centralize the registered layers.

like SEQUENCE_MODEL_LAYERS, etc.